meshnets/data_processing/test_from_hugging_face.py

```diff
@@ -0,0 +1,23 @@
+"""Test Hugging face processing functions."""
+import numpy as np
+import pytest
+
+from .from_hugging_face import to_undirected
```

Comment:
the Python guideline says not to import functions or class names directly... import modules ...

Reply:
this will become: import .data_mappers